src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -44,7 +45,7 @@ type ExternalProps = {
     hasRetentionPolicy?: boolean,
     hasVersions?: boolean,
     onAccessStatsClick?: Function,
-    onClassificationClick: Function,
+    onClassificationClick?: Function,
```

Comment:
Is there a reason why we want to just use Function and not specify what kind of function it could be?

Reply:
for new stuff we are putting all specifics, this is just some dangling legacy stuff

Reply:
i fixed it